.gitignore

```diff
@@ -57,8 +57,3 @@ buck-out/
 
 # CocoaPods
 /ios/Pods/
-
```

Comment:
I am not sure about this... I prefer having this in the gitignore even if this is just a template.

Reply:
I unignored the `.env` files on purpose. Ignoring them can lead to the misconception that these files are a safe place to store secrets but are not. 

These values will be bundled along with the app code making them vulnerable to reverse engineering.

As `react-native-config` docs say: "Keep in mind this module doesn't obfuscate or encrypt secrets for packaging, so do not store sensitive keys in .env. It's basically impossible to prevent users from reverse engineering mobile app secrets, so design your app with that in mind."

For Mobile and SPAs applications, the `.env` files must contain only config values or public keys (many services like google maps API or firebase allow you to specify the bundle_id or URL of your app on the dashboard to ensure they can't be used by others). Secrets should be stored on the backend or a secure vault and retrieved by the app via an HTTPS call if needed.

Reply:
Yup, I know this. That's why I approved the PR.
But I think (maybe I am wrong) that all of our projects have these files ignored, and if that's the case its kind of weird to have the template with the files unignored

Reply:
Okay, I get it, but that's my point. I would prefer new projects to start with these files not ignored. In "Ahead" we made the change and have the files in the repository.